Updating boomkark-bridge commit reference

DT3264 opened the pull request:
Updating boomkark-bridge commit reference

For some reason after pulling to development, sappling insists in including this file, failing otherwise with
```
dt3264@dt3264:~/huronOS/huronOS-build-tools$ sl goto --rev remote/development
abort: unknown revision '61bdb7ad4b0a82df90ded3469b289ec4f09293df'!
```

equetzal commented:
Huh, interesting problem. And this PR fixes this with Sapling? 
Probably we need to document this on the Sapling issues.

DT3264 commented:
In theory it should because otherwise I cannot go directly to development

DT3264 committed:
Updating boomkark-bridge commit reference

For some reason after pulling to development, sappling insists in including this file, failing otherwise with
```
dt3264@dt3264:~/huronOS/huronOS-build-tools$ sl goto --rev remote/development
abort: unknown revision '61bdb7ad4b0a82df90ded3469b289ec4f09293df'!
```
